src/components/Footer.js

```diff
@@ -0,0 +1,11 @@
+export default function Footer() {
+  return (
+    <footer className="App-footer">
```

Comment:
BEM would be: App__footer ;)

Reply:
I was not completely convinced if I should use BEM at this current stage.

Reply:
i would suggest so boris, it will make things lots easier... otherwise you will spend lots of time to renaming stuff... in your html + in your stylesheet too..